doc/latex.rst

```diff
@@ -65,433 +49,487 @@ avoid interpretation as escape sequences, or use raw strings (as is done here).
+      is provided natively by ``'platex'`` and only requires suitable
+      font with ``'xelatex'/'lualatex'``).
 
-     .. code-block:: python
+   .. versionchanged:: 2.3.0
+      ``'xelatex'`` also executes
+      ``\defaultfontfeatures[\rmfamily,\sffamily]{}`` in order to avoid
+      contractions of ``--`` into en-dash or transforms of straight quotes
+      into curly ones in PDF (in non-literal text paragraphs) despite
+      :confval:`smartquotes` being set to ``False``.
 
-         latex_elements = {
-             'packages': r'\usepackage{isodate}'
-         }
+``'textgreek'``
+   This is needed for ``pdflatex`` to support Unicode input of Greek
+   letters such as φύσις.  Expert users may want to load the ``textalpha``
+   package with its option ``normalize-symbols``.
 
-     It defaults to empty.
-
-     The specified LaTeX packages will be loaded before
-     hyperref package and packages loaded from Sphinx extensions.
-
-     .. hint:: If you'd like to load additional LaTeX packages after hyperref, use
-               ``'preamble'`` key instead.
-
-     .. versionadded:: 2.3
-
-  ``'footer'``
-     Additional footer content (before the indices), default empty.
-
-     .. deprecated:: 1.5
-        Use ``'atendofbody'`` key instead.
-
-* Keys that don't need to be overridden unless in special cases are:
-
-  ``'extraclassoptions'``
-     The default is the empty string. Example: ``'extraclassoptions':
-     'openany'`` will allow chapters (for documents of the ``'manual'``
-     type) to start on any page.
-
-     .. versionadded:: 1.2
-     .. versionchanged:: 1.6
-        Added this documentation.
-
-  ``'maxlistdepth'``
-     LaTeX allows by default at most 6 levels for nesting list and
-     quote-like environments, with at most 4 enumerated lists, and 4 bullet
-     lists. Setting this key for example to ``'10'`` (as a string) will
-     allow up to 10 nested levels (of all sorts). Leaving it to the empty
-     string means to obey the LaTeX default.
+   .. hint::
 
-     .. warning::
-
-        - Using this key may prove incompatible with some LaTeX packages
-          or special document classes which do their own list customization.
+      Unicode Greek (but no further Unicode symbols) in :rst:dir:`math`
+      can be supported by ``'pdflatex'`` from setting this key to
+      ``r'\usepackage{textalpha,alphabeta}'``.  Then ``:math:`α``` (U+03B1)
+      will render as :math:`\alpha`.  For wider Unicode support in math
+      input, see the discussion of :confval:`latex_engine`.
 
-        - The key setting is silently *ignored* if ``\usepackage{enumitem}``
-          is executed inside the document preamble. Use then rather the
-          dedicated commands of this LaTeX package.
+   With ``'platex'`` (Japanese),  ``'xelatex'`` or ``'lualatex'``, this
+   key is ignored.
 
-     .. versionadded:: 1.5
+   :Default: ``'\\usepackage{textalpha}'`` or ``''`` if ``fontenc`` does not
+       include the ``LGR`` option.
```

Comment:
Addition to this, the empty setting is used for non-pdflatex engines. see `ADDITIONAL_SETTINGS`.